src/app/containers/manuscript/index.tsx

```diff
@@ -1,33 +1,29 @@
-import React, {useCallback} from 'react';
-import {useDispatch, useSelector} from 'react-redux';
-import {Backdrop, CircularProgress, IconButton} from "@material-ui/core";
-import UndoIcon from '@material-ui/icons/Undo';
-import RedoIcon from '@material-ui/icons/Redo';
+import React, { useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Backdrop, CircularProgress } from '@material-ui/core';
 
-import * as manuscriptActions from "../../actions/manuscript.actions";
-import {canRedoChanges, canUndoChanges, isManuscriptLoaded} from '../../selectors/manuscript.selectors';
+import { isManuscriptLoaded } from '../../selectors/manuscript.selectors';
 import './styles.scss';
-import {ManuscriptEditor} from "./manuscript-editor";
+import { ManuscriptToolbar } from './manuscript-toolbar';
+import { ManuscriptEditor } from './manuscript-editor';
 
-const renderBackdrop = () => (<Backdrop open={true}> <CircularProgress color="inherit" /> </Backdrop>);
+const renderBackdrop = () => (
+  <Backdrop open={true}>
+    {' '}
```

Comment:
I think either linter or prettifier adds these but we need to make it stop :)

Reply:
I don't think it's the linter adding them, nor prettier. I might've missed it whilst rebasing and accidentally restored it. I've removed it for now though...